src/stratigraphy/util/geometric_line_utilities.py

```diff
@@ -171,7 +176,11 @@ def merge_parallel_lines_neighbours(
         if _are_parallel(current_line, line, angle_threshold=angle_threshold) and _are_close(
             current_line, line, tol=tol
         ):
-            current_line = _merge_lines(current_line, line)
+            merged_line = _merge_lines(current_line, line)
```

Comment:
Any idea if this could be solved prettier?

Reply:
Thanks. The change makes sense. I'd just adjust the comment after `if merged_line is not None: # no merge possible`.  to `#  check if merge is possible`. Otherwise it's counterintuitive what happens in the if clause.

I can do the change :).